readme.md

```diff
@@ -401,18 +377,70 @@ With everything else we've seen, Javascript executes one line of code, then when
 
 However, some operations in Javascript are **asynchronous**, meaning Javascript goes on to the next line of code without waiting for the previous line to complete.
 
-This is limited mostly to timing functions, and operations where Javascript is loading data from some other website.
-
-### Why doesn't Javascript wait for these operations to complete before going to the next line of code?
+#### Why doesn't Javascript wait for these operations to complete before going to the next line of code?
 
 Because otherwise the webpage would just "hang" until the operation completes. The browser can't do anything while Javascript is actively running. We've seen this when we've encountered infinite `while` loops. Asynchronicity is a way of preventing the computer from freezing.
 
 This risk is greatest when Javascript is making requests to other webpages. There's no way of knowing how long the request will take to complete. It could be near-instant, but if the target server is having a bad day, it could take who-knows-how-long. You don't want the operability of your computer to be at the mercy of some random computer somewhere else.
 
 In this small app we made, anything we want to be sure happens **after** those 5 seconds of computing should go inside the callback of the `setTimeout`. This way, we can be certain that it will run only when the 5 seconds are up.
 
+## You Do: [TimerJS](https://github.com/ga-wdi-exercises/timer_js) (40 minutes / 1:20)
```

Comment:
Recommend breaking this into a couple chunks. e.g. first 10 minutes is pseudo code and review